docs/Document.md

```diff
@@ -0,0 +1,67 @@
+ # `cardano-shell` overview
+
+`cardano-shell` is an thin layer which brings all the other modules working
+ together and makes sure that they have the required resources 
+ (configuration, exception handling, monitoring, logging, ...).
+
+The shell is also responsible for:
```

Comment:
The shell is not responsible for logging and monitoring. That is brought in using the library, which is a part of shell (the part that can add features).